src/routes/prescricao/edit.js

```diff
@@ -0,0 +1,11 @@
+module.exports = Prescricao => (req, res) => {
+  Prescricao.findOne({
+    where: {
+      id: req.params.prescricao_id
+    }
+  }).then(prescricao => {
+    if(!prescricao) res.render('/404')
```

Comment:
Acho que seria interessante colocar um `return` nesse primeiro `res.render`. A ausência de return aqui vai fazer com que os dois `render` executem, provavelmente causando um erro (algo do tipo `Can't modify the headers after they are sent`).